.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,70 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
+          - url
+
+    steps:
+      - uses: actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc  # v4.1.7
+        with:
+          persist-credentials: false
+
+      - name: Update WPT for subsystem ${{ matrix.subsystem }}
+        run: |
+          git node wpt update --subsystem=${{ matrix.subsystem }}
+        env:
+          GH_TOKEN: ${{ secrets.GH_USER_TOKEN }}
+
+      - name: Run WPT tests
+        id: run-wpt-tests
+        run: |
+          git node wpt run --subsystem=${{ matrix.subsystem }} || echo "WPT tests failed" > wpt-failure.txt
```

Comment:
I can't figure out where `git node wpt run --subsystem` is coming from either.

I think running the WPTs in the action has become redundant since this will open a PR regardless of the outcome. That PR will run github CI straight away and will need full CI afterwards too.

Reply:
I removed the part that runs the WPT Tests as you said